site/content/api/Event-API.md

```diff
@@ -9,8 +9,13 @@ The *Event* API is used by *Devices* to send event messages downstream.
 
 The Event API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using AMQP 1.0 in order to invoke operations of the API as described in the following sections. Throughout the remainder of this page we will simply use *AMQP* when referring to AMQP 1.0.
 
-The *Event* API is identical to the [*Telemetry* API]({{< relref "Telemetry-API.md" >}}) regarding the provided operations, the message format and the message flow. 
-However it provides a different quality of service for messages sent to the *Event* endpoint by routing them via a persistent message broker to guarantee *AT LEAST ONCE* delivery.
+The *Event* API is identical to the [*Telemetry* API]({{< relref "Telemetry-API.md" >}}) regarding the provided operations and the message flow.
+
+The message format for arbitrary events is identical to the [*Telemetry* API]({{< relref "Telemetry-API.md" >}}) as well, but there is a special message format for *lifecycle notifications* which is defined [here]({{< relref "#lifecycle-notifications" >}}).
+
+Events provide a different quality of service for messages sent to the *Event* endpoint by routing them via a persistent message broker 
+to guarantee *AT LEAST ONCE* delivery. 
+*Lifecycle notifications* may be decided to not being persisted, but this is left to the setup of the *AMQP network* and is not further specified in the following.
```

Comment:
This seems to be an issue worth discussing based on my comment above. We currently mark *all* events as *durable* btu I agree that we might want to relax this requirement and allow the issuer of the event to mark it as such or not.